input/elasticapm/docs/spec/v2/transaction.json

```diff
@@ -199,7 +199,13 @@
               "type": [
                 "null",
                 "object"
-              ]
+              ],
+              "additionalProperties": {
+                "type": [
+                  "null",
+                  "string"
+                ]
+              }
```

Comment:
Does this mean we'll ignore cookies that have non-string values? That would also be a breaking change, wouldn't it?

Reply:
What is the current behaviour? Are non-string cookies accepted and then internally mapped to/handled as strings? I'd agree with @axw that this is a breaking change in behaviour then. 
Tbh I believe the switch to enforcing non-nil elements was borderline. With how big of a cost would backwards compatibility come here?

Reply:
By default we were not _indexing_ those fields (we were ingesting them!), but they are still present in the Elasticsearch documents. So a user could still inspect them in the document source, and they could have customised mappings to index them.